### PR TITLE
README: Link Contributors

### PR DESCRIPTION
**Pull request by DeathlyPlays**
This is more or less a proposal, to promote the contributors of the server a bit more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,43 +140,44 @@ Credits
...
 
 Retired Staff
 
-- Bill Meltsner [bmelts] - Development, Sysadmin
-- Cathy J. Fitzpatrick [cathyjf] - Development, Sysadmin
-- Hugh Gordon [V4] - Research (game mechanics), Development
-- Juanma Serrano [Joim] - Development, Sysadmin
+- [Bill Meltsner](https://github.com/bmelts) - [bmelts] - Development, Sysadmin
+- [Cathy J. Fitzpatrick](https://github.com/cathyjf) - [cathyjf] - Development, Sysadmin
+- Hugh Gordonc - [V4] - Research (game mechanics), Development
```
Comment (Ridaz) [inline README.md]: Random c

File: README.md [hunk 2]
```diff
@@ -140,43 +140,44 @@ Credits
...
+- Chris Monsanto - [chaos] - Sysadmin
+- [Leonardo Julca](https://github.com/Slayer95) - [Slayer95] - Development
+- [Mathieu Dias-Martins](https://github.com/Marty-D) - [Marty-D] - Research (game mechanics), Development
+- [The Immortal](https://github.com/TheImmortal) - Development
 
 Retired Staff
 
-- Bill Meltsner [bmelts] - Development, Sysadmin
-- Cathy J. Fitzpatrick [cathyjf] - Development, Sysadmin
-- Hugh Gordon [V4] - Research (game mechanics), Development
```
Comment (AustinXII) [inline README.md]: [Huge Gordon] 
(V4Victini)? He does have a github

File: README.md [hunk 3]
```diff
@@ -140,43 +140,44 @@ Credits
...
-- Guangcong Luo [Zarel] - Development, Design, Sysadmin
+- [Guangcong Luo](https://github.com/Zarel) - [Zarel] - Development, Design, Sysadmin
 
 Staff
 
-- Chris Monsanto [chaos] - Sysadmin
-- Leonardo Julca [Slayer95] - Development
-- Mathieu Dias-Martins [Marty-D] - Research (game mechanics), Development
-- [The Immortal] - Development
+- Chris Monsanto - [chaos] - Sysadmin
```
Comment (AustinXII) [inline README.md]: [Chris Monsanto] (he also has a GitHub) [Chaos]

File: README.md (hunk 2 above)
Comment (DeathlyPlays) [reply README.md]: Alright I'll look, didn't see them when I looked

Comment (DeathlyPlays) [reply README.md]: Don't see it

File: README.md (hunk 3 above)
Comment (DeathlyPlays) [reply README.md]: Found his

File: README.md (hunk 2 above)
Comment (AustinXII) [reply README.md]: He's https://github.com/DanUgore

**Comment by Zarel** [comment]
...no

**Pull request closed without merging**